map/set-max-extent/src/main/java/com/esri/samples/set_max_extent/SetMaxExtentSample.java

```diff
@@ -0,0 +1,162 @@
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.ColorUtil;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+
+public class SetMaxExtentSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      var stackPane = new StackPane();
+      var scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Set Max Extent Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the streets basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create an envelope with an extent covering the state of Colorado
+      Point coloradoNorthWestPoint = new Point(-12139393.2109, 5012444.0468, SpatialReferences.getWebMercator());
+      Point coloradoSouthEastPoint = new Point(-11359277.5124, 4438148.7816, SpatialReferences.getWebMercator());
+      var envelope = new Envelope(coloradoNorthWestPoint, coloradoSouthEastPoint);
+
+      // create a new graphics overlay and add a new graphic to it that shows the Colorado border as a red dashed line
+      var graphicsOverlay = new GraphicsOverlay();
+      graphicsOverlay.getGraphics().add(new Graphic(envelope));
+      var simpleRenderer = new SimpleRenderer(new SimpleLineSymbol(SimpleLineSymbol.Style.DASH,
+        ColorUtil.colorToArgb(Color.RED), 5));
+      graphicsOverlay.setRenderer(simpleRenderer);
+
+      // add the graphics overlay to the map view
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a control panel for toggling the max extent setting
+      var controlsVBox = new VBox(6);
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255,255,255,0.5)"),
```

Comment:
I do like this white background with black label: it's more modern. We will be modernising samples but for now for keeping samples consistent, let's stick with the dark grey background and white label text

`checkbox.setTextFill(Color.WHITE);`

```suggestion
      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.5)"),
```